web/profiles/sdd/sdd.profile

```diff
@@ -7,17 +7,65 @@
  * Copied from https://www.drupal.org/project/multilingual_demo project.
  */
 
+use Drupal\shortcut\Entity\Shortcut;
+use Drupal\user\Entity\User;
+use Drupal\user\RoleInterface;
 use Symfony\Component\Yaml\Parser;
 
 /**
  * Implements hook_install_tasks().
+ *
+ * @see https://www.drupal.org/project/drupal/issues/2982052
  */
 function sdd_install_tasks(&$install_state) {
   return [
+    'sdd_install_setup' => [
+      'display_name' => t('Install setup'),
+      'type' => 'normal',
+    ],
     'sdd_install_import_language_config' => [],
   ];
 }
 
+/**
+ * Performs actions to set up the site for this profile.
+ */
+function sdd_install_setup() {
+  $config_factory = \Drupal::configFactory();
+
+  // Disable the user pictures on nodes.
+  $config_factory->getEditable('system.theme.global')
+    ->set('features.node_user_picture', FALSE)
+    ->save(TRUE);
+
+  // Disable field prefix.
+  $config_factory->getEditable('field_ui.settings')
+    ->set('field_prefix', '')
```

Comment:
Maybe put it into config?

Reply:
Moved to config along with other settings and shortcut as default content